.storybook/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+const webpack = require("webpack");
+const {plugins, resolve} = require("../webpack.common");
+const path = require("path");
+
+module.exports = {
+  resolve,
+  module: {
+    loaders: [
+      {test: /\.json$/, loader: "json"},
+      {
+        test: /\.css?$/,
+        loaders: ['style', 'raw'],
+        include: path.resolve(__dirname, '../')
```

Comment:
OK, did a bit more poking around after Pizza Friday(tm).

I think we'll need to fix this using a few minor tweaks:

1. Update ESLint to **eslint@latest** (which I believe is expertly handled in #1049)

1. Add `!.*` to the .eslintignore, so we don't automatically ignore the files starting with a dot.

1. Explicitly lint the .storybook/ directory by tweaking yamscripts.yml (and in turn package.json) to something like:

  ```yaml
lint: eslint --ext=.js,.jsx,.json . .storybook && sass-lint -v -q
```

---

After those hoops, I get the following output:

```sh
/Users/pdehaan/dev/tmp/del/activity-storybook/.storybook/config.js
  1:29  error  Strings must use doublequote  quotes
  6:11  error  Strings must use doublequote  quotes
  7:11  error  Strings must use doublequote  quotes

/Users/pdehaan/dev/tmp/del/activity-storybook/.storybook/webpack.config.js
   2:7   error  'webpack' is defined but never used  no-unused-vars
  13:19  error  Strings must use doublequote         quotes
  13:28  error  Strings must use doublequote         quotes
  14:42  error  Strings must use doublequote         quotes
```